### DISCUSSION TITLE
When replying, don't make the user avatar clickable

### DISCUSSION THREAD

**Discussion by l-rs** [elk-zone/elk#1652] [Ideas]
When a user replies you get a text box next to your own avatar. However, in the current UI that avatar is clickable. More than once I have accidentally hit the image to be whisked to my profile. (One mini heart attack later) I could return - and Elk remembered I was replying - but there's really no need for the avatar to be clickable/touchable during that action?

**Comment by ZMYaro** [comment]
I am inclined to agree the avatar in the Reply interface should be an indicator of who you are replying as.  We could even one day try to implement the Twitter mobile feature where selecting it opens a drop-down menu that lets you change which of your signed-in accounts you are replying as.
